### ISSUE TITLE
No entities/devices after logging in

### ISSUE THREAD

**Issue by Timmiej93** [myTselection/MijnTuin#6]
I'm getting the error below, and my MijnTuin hub doesn't get any entities or devices. Any clue what's going on?

```
Logger: homeassistant.components.sensor
Source: custom_components/mijntuin/utils.py:79
Integration: Sensor (documentation, issues)
First occurred: September 22, 2023 at 16:27:38 (1 occurrences)
Last logged: September 22, 2023 at 16:27:38

Error while setting up mijntuin platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 359, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/mijntuin/sensor.py", line 79, in async_setup_entry
    await dry_setup(hass, config, async_add_devices)
  File "/config/custom_components/mijntuin/sensor.py", line 43, in dry_setup
    await componentData._force_update()
  File "/config/custom_components/mijntuin/sensor.py", line 111, in _force_update
    gardenLink = await self._hass.async_add_executor_job(lambda: self._session.login(self._username, self._password))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/mijntuin/sensor.py", line 111, in <lambda>
    gardenLink = await self._hass.async_add_executor_job(lambda: self._session.login(self._username, self._password))
                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/mijntuin/utils.py", line 79, in login
    self.gardenlink = self.gardenlink.replace('/calendar','')
                      ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by myTselection** [comment]
Hi, if you navigate to the website https://www.mijntuin.org/dashboard, don't you see a 'Tuinkalender' link in the left side menu?
Maybe you first need to make sure to have several plants added into your collection?
![image](https://github.com/myTselection/MijnTuin/assets/587940/f1746534-af9f-47b8-8895-dd02267b247a)

**Comment by Timmiej93** [comment]
Hi,

Yeah, tuinkalender is there, and it has tasks. Given the error in HA, I suspect it's some kind of login error. <s>I registered with my Google account, does that make a difference? I still had to create a password, which I used to log in in HA.</s>

Well, I guess I'm stupid. Apparently I used the wrong password to log in. After resetting my password on MijnTuin and signing in again, everything now works fine. It would probably be nice to have a more noticeable error for this, but I don't know how easy that is to implement.
